dutch/sms/sms.md

```diff
@@ -3,4 +3,5 @@ Een sms, wanneer krijg je nou nog een sms?
 Het was niet van de overheid, niet van oma. 
 Toch maar klikken.
 En toen kwam de schok...
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.